src/utils/autoBotEngine.js

```diff
@@ -229,6 +232,7 @@ const handleAutoResponse = (commentId, comment, getState) => {
         optionId,
         // add an end property if the optionId is negative
         ...(isEnd ? { end: true } : null),
+        options: chosenOption ? chosenOption.options : null,
```

Comment:
If `undefined` is OK, you can use `chosenOption?.options`.

Otherwise `chosenOption?.options || null`, though this might return `null` for some falsy values of `options`.

Reply:
I think `undefined` is OK in this case as i am testing the value of `options` with 
```
if (options) {
  // here do something with the options array
}
```
So either `undefined` or `null` will do, i will go with your first suggestion i think.